tdrs-backend/tdpservice/users/test/test_api/test_login_redirect_oidc.py

```diff
@@ -0,0 +1,43 @@
+"""Test the LoginRedirectAMS class."""
+from unittest import mock
+from tdpservice.users.api.login_redirect_oidc import LoginRedirectAMS
+
+@mock.patch("requests.get")
+def test_get_ams_configuration(requests_get_mock):
+    """Test the LoginRedirectAMS class."""
+    requests_get_mock.return_value.status_code = 200
+    requests_get_mock.return_value.json.return_value = {"key": "test"}
+    returned_value, error = LoginRedirectAMS.get_ams_configuration()
+    assert returned_value == {'key': 'test'}
+
+    # Test if the configuration is not returned
+    requests_get_mock.return_value.status_code = 500
+    returned_value, error = LoginRedirectAMS.get_ams_configuration()
+    assert returned_value is None
```

Comment:
Im just tagging on to what Andrew was asking/mentioning. Can we also assert on the `error` here since it has more context for the test?